server/webapp/WEB-INF/rails.new/config/routes.rb

```diff
@@ -239,7 +239,10 @@
         resources :packages, param: :package_id, only: [:show, :destroy, :index, :create, :update], constraints: {package_id: ALLOW_DOTS}
         namespace :internal do
           post :material_test, controller: :material_test, action: :test, as: :material_test
-          post :repository_check_connection, controller: :repository_check_connection, action: :check_connection, as: :repository_check_connection
+          controller :package_repository_check_connection do
+            post :repository_check_connection, [action: :repository_check_connection]
+            post :package_check_connection, [action: :package_check_connection]
```

Comment:
Remove the brackets, I provided them in the snippet to indicate they *may* be optional.